### PR TITLE
feat(plugin): rabbitmq plugin

### PR DESCRIPTION
**Pull request by Meyazhagan**
Added RabbitMQ plugin

issue - [#6878](https://github.com/amplication/amplication/issues/6878)

### CODE REVIEW

**Review by abrl91 — DISMISSED** [review]
@Meyazhagan 
Well done!

Just two comments:
1. please make the `RABBITMQ_URLS` visible on the settings. You need to create `.amplicationrc` file (have a look at the postres plugin
2. in the producer service, you have this import: `import { MyMessageBrokerTopics } from "./topics";` if the user changes the name of the broker,  the import will be broken. You need to import based on the name of the broker. You can take it from the context

File: plugins/broker-rabbitmq/webpack.config.js
Comment (overbit) [inline plugins/broker-rabbitmq/webpack.config.js:1]: Please align this configuration with the latest master the plugin-template webpack.config.js

**Review by overbit — CHANGES_REQUESTED** [review]

**Comment by overbit** [comment]
Once done, you can easily request a re-review clicking the button aside the reviewrs
image

**Review by overbit — CHANGES_REQUESTED** [review]

**Comment by overbit** [comment]
Thank you and well done @Meyazhagan!!!
I'm going to merge and publish this plugin soon

**Review by overbit — APPROVED** [review]

**Pull request merged**